client/my-sites/customer-home/locations/primary/index.jsx

```diff
@@ -115,13 +118,14 @@ const Primary = ( { cards, trackCard } ) => {
 	if ( cards.length === 1 ) {
 		isUrgent = urgentTasks.includes( cards[ 0 ] );
 	}
+	const showControlLabels = cards.length !== 1 || isUrgent;
```

Comment:
Why do we want to show dots when there is one urgent task?

Reply:
I don't know! But this was the case before. And this is why I kept it like that. The property was true, whatever the case was, so I kept it like that. I only changed it to match my case.

Reply:
I'm not sure if I understand it correctly. Before, `showControlLabels` was always true. Now, it will be true if we have more than one card (which makes sense), or if we have one card which is urgent.

Reply:
> I'm not sure if I understand it correctly. Before, showControlLabels was always true. Now, it will be true if we have more than one card (which makes sense), or if we have one card which is urgent.

Yes, as you said before it was always true. Either we have an urgent ( which implies that `cards.length === 1` or not (more than one card).

I don't know if it was on purpose, or not, so I left it as is.